src/NHSD.BuyingCatalogue.Solutions.API/Controllers/SolutionsController.cs

```diff
@@ -39,11 +39,11 @@ public SolutionsController(IMediator mediator)
         /// <returns>A task representing an operation to retrieve the details of a Solution.</returns>
         [HttpGet]
         [Route("{id}")]
-        [ProducesResponseType((int)HttpStatusCode.BadRequest)]
+        [ProducesResponseType(typeof(GetSolutionResult), (int)HttpStatusCode.OK)]
```

Comment:
Because we're using the generic `ActionResult<T>` return type, we can simplify this and omit the type argument.

Reply:
Cool, done.